Dockerfile

```diff
@@ -1,29 +1,42 @@
-# Do not update to Ruby 3 until Gemfile dependencies are fixed
-FROM ruby:2.7.6-alpine3.15
+FROM public.ecr.aws/docker/library/ruby:3.2.3-alpine3.19
 
-# These are needed to support building native extensions during
-# bundle install step
-RUN apk --update add --virtual build_deps build-base git
+LABEL org.opencontainers.image.vendor="Ministry of Justice" \
+      org.opencontainers.image.authors="Operations Engineering" \
+      org.opencontainers.image.title="Tech Docs GitHub Pages Publisher" \
+      org.opencontainers.image.description="Container for build and packaging tech-docs" \
+      org.opencontainers.image.url="https://github.com/ministryofjustice/tech-docs-github-pages-publisher"
 
-RUN addgroup -g 1000 -S appgroup \
-  && adduser -u 1000 --system appuser \
-  && adduser appuser appgroup \
-  && gem install bundler -v 2.4.22\
-  && bundle config
+ENV PUBLISHER_DIRECTORY="/publisher" \
+    BUNDLER_VERSION="2.5.6" \
+    LYCHEE_VERSION="0.14.3"
 
-# Required at runtime by middleman
-RUN apk add --no-cache nodejs
 
-WORKDIR /app
+# Install dependencies
+RUN apk --update-cache --no-cache add \
+      build-base \
+      curl \
+      git \
+      nodejs
```

Comment:
Without Lychee we don't need the added guff (git, curl etc)

Reply:
Lychee removed 👋